aws/coreos-docker-servers.tf

```diff
@@ -30,3 +33,23 @@ resource "template_file" "etcd_discovery_url" {
     }
   }
 }
+
+resource "aws_security_group" "docker_node" {
```

Comment:
I wonder if it is worth it to separate the Docker Daemon traffic (port 4243) here. 

Only tsuru API nodes need to talk to the docker daemon, so we could block 4243 for everything but the docker api.